pkg/controller/etcdlockserver/reconcile_members.go

```diff
@@ -114,6 +121,11 @@ func (r *ReconcileEtcdLockserver) reconcileMembers(ctx context.Context, ls *plan
 func memberSpecs(ls *planetscalev2.EtcdLockserver, parentLabels map[string]string) []*etcd.Spec {
 	members := make([]*etcd.Spec, 0, etcd.NumReplicas)
 	for i := 1; i <= etcd.NumReplicas; i++ {
+		// If we're only deploying one member locally, skip the others.
+		if ls.Spec.LocalMemberIndex != nil && int32(i) != *ls.Spec.LocalMemberIndex {
```

Comment:
```suggestion
		if ls.Spec.LocalMemberIndex != nil && (*ls.Spec.LocalMemberIndex != int32(i)) {
```

Reply:
As a matter of consistent style, I always try to write conditional expressions with the dynamic variable first, and the "constant" search value second. In this case, `i` is the value that's changing each time through the loop. `LocalMemberIndex` is a fixed value that we're searching for.